pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -187,9 +192,12 @@ func (cm *containerManagerImpl) getNodeAllocatableAbsolute() v1.ResourceList {
 
 // GetNodeAllocatableReservation returns amount of compute or storage resource that have to be reserved on this node from scheduling.
 func (cm *containerManagerImpl) GetNodeAllocatableReservation() v1.ResourceList {
+	// TODO: When eviction decisions are made on process IDs, if
```

Comment:
We use this function to set `node.Status.Allocatable`, so we don't want to use internalCapacity below

Reply:
Yes, I see...I've created a separate function just for setting node-level cgroups.